Tasks/Bartosh/Task2/Task2.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime
+#dictionaries 
+digits = {1:'один', 2:'два', 3:'три', 4:'четыре', 5:'пять', 6:'шесть', 
+7:'семь', 8:'восемь', 9:'девять', 10:'десять', 11:'одинадцать', 12:'двенадцать', 0:'двенадцать', 
+13:'тринадцать', 14:'четырнадцать', 15:'пятнадцать', 16:'шестнадцать', 17:'семьнадцать', 18:'восемьнадцать', 19:'девятнадцать', 20:'двадцать'}
+
+digits_2 = {2:'двух',  3:'трёх',  4:'четырёх',  5:'пяти',  
+6:'шести',  7:'семи',  8:'восьми',  9:'девяти',  10:'десяти',  
+11:'одинадцати',  12:'двенадцати',  13:'тринадцати',  14:'четырнадцати',  15:'пятнадцати', }
+
+digits_3 = {1:'одна', 2:'две'}
+
+digits_4 = {2:'двадцать', 3:'тридцать', 4:'сорок'}
+
+hours_next = {0:'первого', 1:'второго', 2:'третьего', 3:'четвертого', 4:'пятого', 5:'шестого', 6:'седьмого', 
+7:'восьмого', 8:'девятого', 9:'десятого', 10:'одинадцотого', 11:'двенадцотого'}
+
+#input option selection
+option = int(input('Hello! I can write time by words.\nPress 1, if you want to know current time\nor\npress 2 (or any key), if you want to enter time in hh:mm format\n1/2?:'))
+
+# option 1 for current time
+if option == 1:
+    timestamp = datetime.now()
+    t = str(timestamp.hour)+':'+str(timestamp.minute)
+# 2 (or any key) for input from keyboard
+else:
+    while True:#validation that input data is in time range 
+      t = input ('press time in hh:mm format:')
+      if int(t.split(':')[0])<0 or int(t.split(':')[0])>23 or int(t.split(':')[1])<0 or int(t.split(':')[1])>59:
```

Comment:
what if it does not have a ':'?) you should check this beforehand, validation is very important, -1 point for the unwise choice